src/parser/doesSomeParentMatch.ts

```diff
@@ -1,19 +1,18 @@
-//@ts-nocheck
```

Comment:
this was an easy one.
parentNode was used instead o parentElement because IE i think